struct_env/struct_env.py

```diff
@@ -49,20 +94,33 @@ def reset(self):
         # Choose the agent's belief
         self.time_step = 0
         self.beliefs = self.belief0
+        self.beliefsc = self.belief0c
+        self.alphas = self.alpha0
         self.drate = np.zeros((self.ncomp, 1), dtype=int)
         self.observations = {}
         for i in range(self.ncomp):
             self.observations[self.agent_list[i]] = np.concatenate(
                 (self.beliefs[i], [self.time_step / 30]))
+
+        if self.env_type is "correlated":
+            for i in range(self.ncomp):
+                self.observations[self.agent_list[i]] = np.concatenate(
+                    (self.observations[self.agent_list[i]], self.alphas, [self.time_step / 30]))
```

Comment:
I have also added the normalized timestep here.

Reply:
Nice, missed that one :D

Reply:
Oh I went too quickly here.
I thought you added the normalisation.
In fact timestep is already in the observation, because I append alphas to the observation (belief+timestep abose) when we are in correlated mode.
For ablation study, it will then be easy to choose to add it or not in the observation.

Reply:
[https://github.com/moratodpg/ray_environments/pull/7/commits/443a433a717db4448eb9d2543cb0b0e6cc089df9#diff-df106087ea8381e5153ab17bdf66cdefe4b25af00235ab7cb504fdd43ad5e37eR107](test)